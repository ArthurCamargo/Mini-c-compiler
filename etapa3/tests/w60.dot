digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 [label="a"];
n3;
n3 -> n4;
n3 -> n5;
n4;
n4 -> n6;
n6;
n6 [label="b"];
n4 [label="#"];
n5;
n5 [label="2"];
n3 [label="+"];
n1 [label="="];
n0 [label="f"];

}