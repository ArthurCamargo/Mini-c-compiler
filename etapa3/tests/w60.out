0x55ca0ada5350
0x55ca0ada5350, 0x55ca0ada5650
0x55ca0ada5650
0x55ca0ada5650, 0x55ca0ada54a0
0x55ca0ada5650, 0x55ca0ada5600
0x55ca0ada54a0
0x55ca0ada54a0 [label="a"];
0x55ca0ada5600
0x55ca0ada5600, 0x55ca0ada5560
0x55ca0ada5600, 0x55ca0ada55b0
0x55ca0ada5560
0x55ca0ada5560, 0x55ca0ada5510
0x55ca0ada5510
0x55ca0ada5510 [label="b"];
0x55ca0ada5560 [label="#"];
0x55ca0ada55b0
0x55ca0ada55b0 [label="2"];
0x55ca0ada5600 [label="+"];
0x55ca0ada5650 [label="="];
0x55ca0ada5350 [label="f"];
